tests/general/IndentationBearTest.py

```diff
@@ -151,3 +155,14 @@ def test_unmatched_indents(self):
 
         invalid_file2 = ('{}}\n',)
         self.verify_bear(valid_file=None, invalid_file=invalid_file2)
+
+    def test_multiple_indent_specifiers(self):
```

Comment:
SpaceConsistencyBear likes to eat up those extra leading spaces in my strings, that's why i use, tabs and only use spaces when necessary